tools/checked-c-convert/ConstraintBuilder.cpp

```diff
@@ -150,9 +156,16 @@ class FunctionVisitor : public RecursiveASTVisitor<FunctionVisitor> {
 
       if (SR.isValid() && FL.isValid() && !FL.isInSystemHeader() &&
         (D->getType()->isPointerType() || D->getType()->isArrayType())) {
+        // add the variable with in the function body context.
         Info.addVariable(D, S, Context);
 
         specialCaseVarIntros(D, Info, Context);
+        // if this is a static array declaration.
```

Comment:
Do you mean "if this is an array declaration" or "a statically sized array declaration"..   The term "static array" has special meaning in C: it means an array with static storage

Reply:
Fixed